File: MVC/CodePlexMVCTemplates/SolutionItems/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Title") = "Home Page"
        ViewData("Message") = "Welcome to ASP.NET MVC!"

        Return RenderView()
    End Function

    Function About() As ActionResult
        ViewData("Title") = "About Page"

        Return RenderView()
    End Function
End Class

File: MVC/CodePlexMVCTemplates/SolutionItems/Controllers/HomeController.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace $safeprojectname$.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewData["Title"] = "Home Page";
            ViewData["Message"] = "Welcome to ASP.NET MVC!";

            return RenderView();
        }

        public ActionResult About()
        {
            ViewData["Title"] = "About Page";

            return RenderView();
        }
    }
}